### PR TITLE
logs: better names and comments

### PR DESCRIPTION
**Pull request by gabor**
this pull request has no functional changes, we only change variable-names, function-names and comments, to better explain what's going on in the code.

the current version of the comments calls all "extra fields from the dataframe" "derived fields" (a concept done in Loki), which is only correct when the data comes from Loki. for example, when the data comes from Elasticsearch, those "extra fields from the dataframe" are real data, coming from the database.

i tried to adjust the naming and comments to better explain what's going on.

if you have any idea how to improve it even further, please tell me. thanks!

how to test:
- if you review it and agree that there are no functional changes, not much testing is needed
- maybe just run the explore-page for a logs-datasource, and see if opening the log-details still shows the correct data

### CODE REVIEW

File: packages/grafana-ui/src/components/Logs/logParser.ts [hunk 1]
```diff
@@ -14,16 +14,24 @@ type FieldDef = {
...
   (row: LogRowModel, getFieldLinks?: (field: Field, rowIndex: number) => Array<LinkModel<Field>>) => {
-    const fields = parseMessage(row.entry);
-    const derivedFields = getDerivedFields(row, getFieldLinks);
-    const fieldsMap = [...derivedFields, ...fields].reduce((acc, field) => {
+    const logMessageFields = parseMessage(row.entry);
+    const dataframeFields = getDataframeFields(row, getFieldLinks);
+    const fieldsMap = [...dataframeFields, ...logMessageFields].reduce((acc, field) => {
       // Strip enclosing quotes for hashing. When values are parsed from log line the quotes are kept, but if same
       // value is in the dataFrame it will be without the quotes. We treat them here as the same value.
+      // we need to handle this scenario:
```
Comment (matyax) [inline packages/grafana-ui/src/components/Logs/logParser.ts:27]: One small nit from my part: I would respect the capitalization used in the rest of the file. Absolutely minor, but looks better IMO.

**Review by matyax — APPROVED** [review]
IMO, this is a good improvement and adds relevant context to the existing code 👍

File: packages/grafana-ui/src/components/Logs/logParser.ts (hunk 1 above)
Comment (gabor) [reply packages/grafana-ui/src/components/Logs/logParser.ts]: good point, updated.

**Review by matyax — APPROVED** [review]
👌

**Pull request merged**